doc/classes/TextEdit.xml

```diff
@@ -282,6 +282,13 @@
 				Returns the selection end line.
 			</description>
 		</method>
+		<method name="get_visible_line_count" qualifiers="const">
+			<return type="int">
+			</return>
+			<description>
+				Returns the number of visible lines, including text warped.
```

Comment:
```suggestion
				Returns the number of visible lines, including wrapped text.
```

Reply:
You did another type. Should be w**ra**pped not w**ar**pped

Reply:
Oh, sorry ;-;